src/content/getting-started/observability/logging/disable/index.md

```diff
@@ -0,0 +1,76 @@
+---
+linkTitle: Disable logging
+title: Disable Logging
+description: This article explains how to turn off all logging for an entire workload cluster.
+weight: 50
+menu:
+  main:
+    identifier: getting-started-observability-logging-disable
+    parent: getting-started-observability-logging
+user_questions:
+  - How can I disable logging on my cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2023-10-11
+aliases:
+  - /ui-api/observability/logging/disable
+  - /observability/logging/disable
+---
+
+In this article you will learn how you can disable logging for your cluster.
+
+## Introduction to logging
+
+Each cluster created on the Giant Swarm platform benefits from our logging stack which allow us to retrieve logs from Kubernetes and GiantSwarm components. 
```

Comment:
I would add a disclaimer here because 1, this is the case only on vintage, 2 that's starting from release v19.1.0. Also, it is currently disabled by default